www/src/pages/404.js

```diff
@@ -8,6 +8,12 @@ class FourOhFour extends React.Component {
       <Layout location={this.props.location}>
         <Container>
           <h1>Page not found</h1>
+          <p>
+            Oops! The page you are looking for has been removed or relocated.
+          </p>
+          <a href="../">
```

Comment:
When linking to internal pages you should use gatsby-link. Also the `to` prop then would be `"/"` not `../"`